packages/lib/formulas/formatDate/formula.json

```
@@ -15,12 +15,12 @@
     },
     {
       "name": "Options",
-      "description": "Optional Object for configuring the formatting of the output. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#syntax for all options.",
+      "description": "Optional Object for configuring the formatting of the output.",
```

I think the links are relevant. But let's render them as markdown instead and make sure we do the same in the editor when we show their descriptions?
```suggestion
      "description": "Optional Object for configuring the formatting of the output. See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#syntax) for all options",
```

---

Yes please.